NSTTutorials/UITestCodeGeneration/README.md

```diff
@@ -0,0 +1,74 @@
+- `NAVIGATION/ENTRY` - `NAVIGATION` and `ENTRY` are linked in how to understand their functionality. These, along with IMPORTS and MEMBER_FIELDS, are what would be added / exported to the UI test file itself.
+    
+    `NAVIGATION` and `ENTRY` are dependent on the sequence of service wrapper requests that are sent during the course of the NST test. 
+    
+    ```
+    IF I send a service wrapper request
+    AND the request is the first request that is made within the test method
+    THEN utilize the ENTRY column data as the output to the UI test file
+    ELSE utilize the NAVIGATION column data as the output to the UI test file
+    ```
+    
+    The use case here is that when certain APIs are triggered FIRST, they would be associated with some initial action, such as loading a UI view for the first time - whereas any API request AFTER the initial call would be associated with performing some action AFTER the initial view has been loaded - hence the NAVIGATION name of the column.<br><br>
+    
+- `IMPORTS` - Any imports that may be required by the `NAVIGATION` and `ENTRY` columns for that row (e.g. Java package imports).<br><br>
+- `MEMBER_FIELDS` - Any member fields that may be required by the `NAVIGATION` and `ENTRY` columns for that row (e.g. instantiation of shared UI test classes).
+
+## Utilizing the iOS & Android loggers
+By default, NST comes with UI test code generation classes specifically built for iOS (`.swift`) & Android (`.java`) output. As we saw above, we will need to ensure that the `nstToFuiMappings` file is set up for either iOS or Android (`nstToFuiMappingsIos`/`nstToFuiMappingsAndroid`) in our resources directory.
+
+As long as we have the appropriate mapping CSV file set up, then the default loggers for either iOS or Android will be utilized. To summarize their implementation, they will look for the classes contained in the `androidTestsLocation` / `iosTestsLocation` (depending on platform) that match both the **class name** and the **method name** of the NST test that is being executed. Once it finds those, it will map the output using the output mapping CSV to the UI test files / classes.
+
+To see an example of this in action, run the test method in the [UITestCodeGenerationTest](src/test/java/com/ebay/nst/tutorials/rest/uitestcodegeneration/UITestCodeGenerationTest.java) class, and observe the UI test code that is generated in the [UITestCodeGenerationTest.swift](src/test/java/com/ebay/nst/tutorials/rest/uitestcodegeneration/UITestCodeGenerationTest.swift) class.
+Additionally, you can modify the mapping CSV file to see the changes reflected in the associated Swift class (after re-running the NST test).
+
```

Comment:
Is it worth having a paragraph that explains different approaches to problems one may encounter around having 'flexible' code generated? For example, adding statements that take parameters gives the test writer the flexibility to mutate a behavior in generated code by setting parameter values.

Reply:
Done in 3d43f0926bdcdf8d1cddbe7d5466662922cf51bb